### DISCUSSION TITLE
Enforce rule 1.5 in world chat some more

### DISCUSSION THREAD

**Discussion by krutoj87** [chromiecraft/chromiecraft#7394] [Suggestions]
Lately it seems like the political talk in world chat gets out of hand. Can you please enforce the rule 1.5 in world chat some more.

It's probably not possible to have a mod around 24/7. But maybe a mod can log in once in a while and give timeouts to people who talk about political or religious topics in world chat.

**Comment by BeefSteaker** [comment]
Hi, as you mentioned staff can't be active 24/7 so we sometimes miss stuff. If you see anything feel free to open a ticket on discord and we can deal with it

**Comment by Amrx2** [comment]
Just to confirm, we have handed out 5 mutes and 6 warnings in the last 3 days as well. 

Anything not caught by the team, as you mention we cannot be on 24/7, please make a report ticket on discord. Cheers!

**Reply by krutoj87** [reply]
Maybe consider adding a weekly statistic like that to the mod log on discord. 
As players the work you mods do is often invisible to us. So if we see people breaking the rules in world chat, it sometimes feels like no actions are taken against it, when in fact we just don't see the actions. 
With a weekly statistic it would make the work visible to us players. It would show people that rules get enforced, which is reassuring, and hopefully makes people think twice before breaking rules.

And I will also make sure to make screenshots and report that stuff in the future, because in the past I only used the report function ingame. But I'm not sure if this function does anything more than just muting the player for me.

**Reply by Amrx2** [reply]
> Maybe consider adding a weekly statistic like that to the mod log on discord. As players the work you mods do is often invisible to us. So if we see people breaking the rules in world chat, it sometimes feels like no actions are taken against it, when in fact we just don't see the actions. With a weekly statistic it would make the work visible to us players. It would show people that rules get enforced, which is reassuring, and hopefully makes people think twice before breaking rules.
> 
> And I will also make sure to make screenshots and report that stuff in the future, because in the past I only used the report function ingame. But I'm not sure if this function does anything more than just muting the player for me.

We have the mod log what captures anything that has lead to 24 hours or more of action taken against the account. Its a time thing more than anything, with our resource being taken up with other tasks hence the bar to hit to enter the modlog.

The ingame report function only ignores people on your end, please make sure you are reporting it via the 'report player' ticket in discord, thanks.
